ocs_ci/deployment/deployment.py

```diff
@@ -984,7 +985,8 @@ def setup_local_storage(storageclass):
             "Creating optional operators CatalogSource and ImageContentSourcePolicy"
         )
         run_cmd(f"oc create -f {optional_operators_yaml.name}")
-
+        # sleep here to start update machineconfigpool status
+        time.sleep(60)
```

Comment:
I would transform comment above to log message:
logger.info("sleep for 60 sec to start update machineconfigpool status")

Reply:
removed the cherry-picked comment which was causing this